### DISCUSSION TITLE
How do I clear this status message?

### DISCUSSION THREAD

**Discussion by n0kovo** [Textualize/rich#1000] [Q&A]
Hi. 

First of all, this library is amazing, thank you @willmcgugan!

I am trying to clear a status message that was created in a `with` statement from another function, after the function is interrupted, but I can't seem to figure out how.

I've made a little demo below. After pressing Ctrl+C, the status message is still visible until `sys.exit(0)`.
It would be nice if I could just do something like `console.status.clear()`

Any suggestions?

```python
from rich.console import Console
import signal
import sys
from time import sleep

console = Console()

def signal_handler(sig, frame):
    # This is where I would like the status to disappear
    console.print("Ctrl+C (SIGINT) caught.")
    console.print("The status message is still visible here.")
    console.print("How do I make it go away?")
    console.print("Sleeping for 5 sec.")
    sleep(5)
    sys.exit(0)

# Call signal_handler() when receiving SIGINT (Ctrl+C)
signal.signal(signal.SIGINT, signal_handler)

# Infinite loop with status message
with console.status("Waiting for Ctrl+C") as wait_status:
        while True:
            console.print("Press Ctrl+C")
            sleep(1)
```

**Comment by willmcgugan** [comment]
I generally prefer catching the KeyboardInterrupt exception which will allow the context manager clear the status. But if you want to do it in a signal handler you could try `wait_status.stop()`
